.travis.yml

```diff
@@ -1,18 +1,29 @@
 language: python
 
+sudo: "required"
+
 python:
-  - "2.7"
-  - "3.5"
+    - "2.7"
+    - "3.5"
 
-sudo: "required"
+env:
+  global:
+    - DISPLAY=:99.0
+  matrix:
+    - PYTHON_2=2.7
+    - PYTHON_3=3.5
```

Comment:
I think switching to tox-travis (see other comment) would mean we can remove this matrix.